src/common/hooks/useCancelOrder/useSendOnChainCancellation.ts

```diff
@@ -16,29 +17,33 @@ export function useSendOnChainCancellation() {
   const addTransaction = useTransactionAdder()
   const getOnChainCancellation = useGetOnChainCancellation()
 
-  return useCallback(
-    async (order: Order) => {
-      if (!chainId) {
-        return
-      }
+  const processCancelledOrder = useCallback(
```

Comment:
With ComposableCow orders we might cancel more than one order, hence, we need to process it more that once